docs/index.rst

```diff
@@ -16,8 +16,6 @@ Contents:
    getting_started
    testing
    internationalization
-   modules
-   changelog
```

Comment:
FYI - I've made similar changes in my PR: https://github.com/edx/edx-developer-docs/pull/5

Reply:
Sounds good.  We may have a little conflict to resolve, but it will be simple to keep your changes.  It would nice if this PR can be self-consistent.  If you feel otherwise, I'll revert.